.storybook/pages/AlongDemo/AlongDemo.tsx

```diff
@@ -0,0 +1,479 @@
+import clsx from 'clsx';
+import React, { useState } from 'react';
+import { useUID } from 'react-uid';
+
+import styles from './AlongDemo.module.css';
+import globalStyles from './GlobalStyles.module.css';
+
+import {
+  Button,
+  ButtonGroup,
+  Dropdown,
+  Heading,
+  Icon,
+  Label,
+  Link,
+  Tabs,
+  Tab,
+  Text,
+  Tooltip,
+} from '../../../src';
+
+import AlongFooterPaper from '../../static/along-footer-paper.svg';
+import AlongLogoBulb from '../../static/along-logo-bulb.svg';
+import GoogleLogo from '../../static/google-logo.svg';
+import MicrosoftLogo from '../../static/microsoft-logo.svg';
+import PlaceholderImage from '../../static/placeholder-image.svg';
+import Sprout from '../../static/sprout.svg';
+
+const GlobalFooter = ({ className }: { className?: string }) => (
+  <div className={clsx(styles['wireframe-demo__footer'], className)}>
+    <Link>
+      Privacy Policy{' '}
+      <Icon
+        name="open-in-new"
+        purpose="informative"
+        size="0.8em"
+        title="opens in a new tab"
+      />
+    </Link>
+    <Link>Cookie Settings</Link>
+    <Link>
+      User Agreement{' '}
+      <Icon
+        name="open-in-new"
+        purpose="informative"
+        size="0.8em"
+        title="opens in a new tab"
+      />
+    </Link>
+    <Link>
+      Code of Conduct{' '}
+      <Icon
+        name="open-in-new"
+        purpose="informative"
+        size="0.8em"
+        title="opens in a new tab"
+      />
+    </Link>
+    <Link>
+      Help Center{' '}
+      <Icon
+        name="open-in-new"
+        purpose="informative"
+        size="0.8em"
+        title="opens in a new tab"
+      />
+    </Link>
+  </div>
+);
+
+const LoggedOutPage = ({ onLogin }: { onLogin: () => void }) => (
+  <div className={styles['logged-out-page']}>
+    <div className="flex flex-col items-center mb-16 p-8 grow">
+      <header className="flex gap-4">
+        <img
+          alt="placeholder for decorative illustration"
+          className={styles['logged-out-page__header-img']}
+          src={PlaceholderImage}
+        />
```

Comment:
This is still using placeholder images because I actually don't know how to import a non-svg image in EDS..